client/ayon_core/plugins/publish/collect_anatomy_instance_data.py

```diff
@@ -399,6 +399,10 @@ def _fill_folder_data(self, instance, project_entity, anatomy_data):
                 "parent": parent_name,
```

Comment:
Shouldn't this just be calling the function from `ayon_core.pipeline.template_data.get_folder_template_data`?

```python
data = get_folder_template_data(folder_entity)
anatomy_data.update(data)
```